packages/strapi-admin/admin/src/utils/FieldApi.js

```diff
@@ -0,0 +1,36 @@
+import { cloneDeep } from 'lodash';
+import invariant from 'invariant';
+
+class FieldApi {
+  fields = {};
+
+  getField = type => {
+    invariant(type, 'A type must be provided');
+
+    return cloneDeep(this.fields[type]) || null;
+  };
+
+  getFields = () => {
```

Comment:
Maybe this function should be an es6 `getter` 
```js
get fields() {
    return cloneDeep(this._fields);
}
```

Then, you will be able to get fields without call the function `strapi.FieldAPI.fields`.